CanonicalLtd/landscape-client
# Implicitly trust file-local sources managed by landscape (LP: #1736576)

Testing instructions:

* install landscape-client on artful+ and register
* apply a simple package profile (depends sl)
* notice failure to update packages (Release file not found)
* apt-get build-depends ./ && dpkg-buildpackage -b --us -uc && dpkg -i ../*.deb
* check upgrade added trust option to :
  /etc/apt/sources.list.d/_landscape-internal-facad.list[.orig]
* re-check "apt-get update" works
